test/resources/knativeresources.go

```diff
@@ -177,10 +169,21 @@ func IsKnativeObsoleteResourceGone(clients *test.Clients, namespace string, obsR
 		gvr := apis.KindToResource(resource.GroupVersionKind())
 		var err error
 		if resource.GetNamespace() != "" {
-			// This is a namespaced resource
+			// Verify all namespaced resources, except jobs.
```

Comment:
Why not Jobs?

Reply:
We added a job transformation, so the job name won't match the one of the launched job. It is meaningless to check jobs.

Reply:
OK, so long as we don't create a new job every reconcile loop, I suppose.

I wouldn't say it's meaningless to check jobs, though. They may be necessary for the correctness of the upgrade.